modules/router/src/router.proto

```diff
@@ -33,6 +34,31 @@ message SocketData {
   repeated string rooms = 4;
 }
 
+message ProxyRouteOptions {
```

Comment:
proxy routes should also include an "action"/"method" option so that you can define when this proxy is active.

In contrast with regular routes, this method option should also include the "ALL" value, where the path would target all methods. So valid values would be GET, POST, PUT, PATCH, DELETE, ALL